test/api_getProductRaw_test.dart

```diff
@@ -64,9 +50,9 @@ void main() {
       expect(result.product!.nutriments!.fat, 9.2);
       expect(result.product!.nutriments!.saturatedFat, 1.1);
       expect(result.product!.nutriments!.proteins, 4.5);
-      expect(result.product!.nutriments!.novaGroup, 4);
+      expect(result.product!.nutriments!.novaGroup, null);
```

Comment:
Does this mean that we had a score but now with the new data it is gone

Reply:
It seems like it, yes.